docs/singularity/02.singularity_batch.rst

```diff
@@ -25,11 +25,11 @@ On a HPC system, your job submission script would look something like:
 .. code-block:: bash
 
   #!/bin/bash
-  
+
   #SBATCH -J myjob                      # Job name
   #SBATCH -o output.%j                  # Name of stdout output file (%j expands to jobId)
-  #SBATCH -p normal                     # Queue name
-  #SBATCH -N 1                          # Total number of nodes requested (68 cores/node)
+  #SBATCH -p small                      # Queue name
```

Comment:
we need to check with Lauren if the 'small' queue is the one she is asking for the reservation in

Reply:
I was wondering that as well (or if we'd have a reservation). In terms of cutting & pasting from the docs after the class, I figured either the 'small' or 'flex' queue made the most sense... 'development' is frequently full and the 'normal' queue requires a minimum of 3 nodes.